src/assignments/bup/components/adios.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import autobind from 'class-autobind'
+
+class Adios extends React.Component {
+
+    constructor() {
+        super(...arguments)
+        autobind(this)
+    }
+    render() {
+        const { ...props } = this.props
```

Comment:
Hey Dave, this can be written as `const {first, last} = this.props` to pull out new variables, or just use `this.props.first`. This line is technically the same as `const props = this.props`